getting_started/first_2d_game/05.the_main_game_scene.rst

```diff
@@ -337,11 +339,11 @@ Note that a new instance must be added to the scene using ``add_child()``.
         mob.position = mob_spawn_location.position
 
         # Add some randomness to the direction.
-        direction += rand_range(-PI / 4, PI / 4)
+        direction += randi_range(-PI / 4, PI / 4)
```

Comment:
Did you test this / are you sure this is right? According to https://github.com/godotengine/godot/pull/40718, the 4.0 equivalent of `rand_range()` is `randf_range()`, not `randi_range()`. Notably, `randi_range()` uses `int` instead of `float`, so we may lose a lot due to rounding here compared to the old behaviour.

Reply:
I did test, but I didn't know what it should look like... oops.  It is better with randf_range (otherwise it's just 4 directions..).  Updated with changes